piptools/repositories/pypi.py

```diff
@@ -37,21 +37,7 @@
 )
 from .base import BaseRepository
 
-from piptools._compat.pip_compat import PIP_VERSION
-
-try:
-    from pip._internal.req.req_tracker import RequirementTracker
-except ImportError:
-
-    @contextmanager
-    def RequirementTracker():
-        yield
-
-
-try:
-    from pip._internal.cache import WheelCache
-except ImportError:
-    from pip.wheel import WheelCache
+from piptools._compat.pip_compat import PIP_VERSION, RequirementTracker, WheelCache
```

Comment:
Could you convert it to the relative import? It seems the whole codebase in `piptools` follows that rule.
<sup>Though, I'd prefer absolute import.</sup>